src/modules/account/components/AccountRow/AccountRow.test.js

```diff
@@ -8,7 +8,7 @@ jest.mock('react-i18next');
 
 const props = {
   account: mockSavedAccounts[0],
-  showRemove: true,
+  onRemove: jest.fn(),
```

Comment:
@oskarleonard Use a constant for the onRemove mock.
For example, `const mockOnRemove = jest.fn() and then reuse it here and while rendering the component below
```suggestion
  onRemove: mockOnRemove,
```